### ISSUE TITLE
Additional Strange CSS Unicode Problems

### ISSUE THREAD

**Issue by bmuenzenmeyer** [pattern-lab/styleguidekit-assets-default#96]
![image](https://user-images.githubusercontent.com/298435/31921907-7af3ee74-b837-11e7-8864-9a5292281188.png)

![image](https://user-images.githubusercontent.com/298435/31921912-7fafc064-b837-11e7-8ab5-11b52087ef86.png)

**Comment by bradfrost** [comment]
Very strange! The right-facing arrow makes sense, but the block-level comment shouldn't be doing that. Will take a look.

**Comment by bradfrost** [comment]
@bmuenzenmeyer for these changes, what branch should I be working in? I want to make sure I'm on the straight and narrow.

**Comment by bmuenzenmeyer** [comment]
@bradfrost branching off `develop` works just fine.

I tried to document this within https://github.com/pattern-lab/styleguidekit-assets-default/blob/master/.github/CONTRIBUTING.md#guidelines - let me know if you think we could improve that anywhere

**Comment by bradfrost** [comment]
@bmuenzenmeyer The contributing guidelines are good. I guess I'm trying to figure out if this small change would necessitate its own branch, or could that work happen right on `develop`? That's typically how I operate with git flow, but just want to make sure I'm following conventions.

**Comment by frostyweather** [comment]
@bmuenzenmeyer Is this within `dist` that you are seeing this? I am not seeing it on my pulled down version of this.

**Comment by bmuenzenmeyer** [comment]
@frostyweather yes I see it in dist. I suspect it only manifests on Windows, since you guys haven't had problems yet right?

**Comment by frostyweather** [comment]
@bmuenzenmeyer I have not had problems. Must be a windows thing. Maybe this sort of issue. https://stackoverflow.com/questions/44334638/css-content-character-rendering-strange

**Comment by frostyweather** [comment]
Changed the arrow to Unicode value in CSS within the `feature-pattern-info-panel` branch. Hopefully this fixes this issue
